### DISCUSSION TITLE
How to enable word wrap on startup?

### DISCUSSION THREAD

**Discussion by Niiisse** [syl20bnr/spacemacs#15398] [Q&A] [answered]
Hey,

I'd like to enable `toggle-word-wrap` always, but adding it to the user-config part of the .spacemacs file doesn't do anything. I've tried a bunch of variations but I really can't seem to figure it out. I'm pretty new to Spacemacs, so forgive me if it's an obvious thing I'm missing.

How do I run built-in functions on startup? Thanks in advance.

**Comment by tshu-w** [comment, marked as answer]
For `word-wrap`, you should set it directly `(setq-default word-wrap t)`

**Reply by Niiisse** [reply]
In `user-config`?

**Reply by tshu-w** [reply]
@Niiisse Yes, or `user-init`

**Reply by Niiisse** [reply]
It works, thank you!
